packages/governance/exported.d.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable -- doesn't understand .d.ts */
+/**
+ * @file re-export types into global namespace, for consumers that expect these
+ *   to be ambient
+ */
+
+// export everything
+export * from './src/types.js';
+
+// XXX re-export types into global namespace, for consumers that expect these to
+//  be ambient. Why the _ prefix? Because without it TS gets confused between the
+//  import and export symbols. h/t https://stackoverflow.com/a/66588974
+//  Note one big downside vs ambients is that these types will appear to be on `globalThis`.
```

Comment:
fair price, given that this is a transitional measure.

Is it worth an UNTIL comment?